### ISSUE TITLE
Fail on startup with purpur 1.20.6

### ISSUE THREAD

**Issue by trentnbauer** [ShatteredSoftware/ShatteredCore#30]
```
[14:16:45 INFO]: [ShatteredScrolls] Enabling ShatteredScrolls v2.0.0 (fb1a98d)
[14:16:45 WARN]: [ShatteredScrolls] Config invalid or not found. Generating a default one.
[14:16:46 INFO]: [ShatteredScrolls] Loading recipes.
[14:16:46 ERROR]: Error occurred while enabling ShatteredScrolls v2.0.0 (fb1a98d) (Is it up to date?)
java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
        at ShatteredCore-1.5.0.jar/com.github.shatteredsuite.core.include.nbt.utils.MinecraftVersion.getVersion(MinecraftVersion.java:78) ~[ShatteredCore-1.5.0.jar:?]
        at ShatteredCore-1.5.0.jar/com.github.shatteredsuite.core.include.nbt.NBTCompound.getType(NBTCompound.java:713) ~[ShatteredCore-1.5.0.jar:?]
        at ShatteredCore-1.5.0.jar/com.github.shatteredsuite.core.include.nbt.NBTCompound.addCompound(NBTCompound.java:557) ~[ShatteredCore-1.5.0.jar:?]
        at ShatteredScrolls-LATEST-[2.0.0].jar/com.github.shatteredsuite.scrolls.items.ScrollInstance.applyNBT(ScrollInstance.kt:32) ~[ShatteredScrolls-LATEST-[2.0.0].jar:?]
        at ShatteredScrolls-LATEST-[2.0.0].jar/com.github.shatteredsuite.scrolls.items.ScrollInstance.toItemStack(ScrollInstance.kt:68) ~[ShatteredScrolls-LATEST-[2.0.0].jar:?]
        at ShatteredScrolls-LATEST-[2.0.0].jar/com.github.shatteredsuite.scrolls.recipe.RecipeHandler.registerRecipes(RecipeHandler.kt:15) ~[ShatteredScrolls-LATEST-[2.0.0].jar:?]
        at ShatteredScrolls-LATEST-[2.0.0].jar/com.github.shatteredsuite.scrolls.ShatteredScrolls.loadContent(ShatteredScrolls.java:168) ~[ShatteredScrolls-LATEST-[2.0.0].jar:?]
        at ShatteredScrolls-LATEST-[2.0.0].jar/com.github.shatteredsuite.scrolls.ShatteredScrolls.postEnable(ShatteredScrolls.java:143) ~[ShatteredScrolls-LATEST-[2.0.0].jar:?]
        at ShatteredCore-1.5.0.jar/com.github.shatteredsuite.core.ShatteredPlugin.onEnable(ShatteredPlugin.java:110) ~[ShatteredCore-1.5.0.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:287) ~[purpur-api-1.20.6-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202) ~[purpur-1.20.6.jar:git-Purpur-2199]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109) ~[purpur-1.20.6.jar:git-Purpur-2199]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:519) ~[purpur-api-1.20.6-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:623) ~[purpur-1.20.6.jar:git-Purpur-2199]
        at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:572) ~[purpur-1.20.6.jar:git-Purpur-2199]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:684) ~[purpur-1.20.6.jar:git-Purpur-2199]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:443) ~[purpur-1.20.6.jar:git-Purpur-2199]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:357) ~[purpur-1.20.6.jar:git-Purpur-2199]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1156) ~[purpur-1.20.6.jar:git-Purpur-2199]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:329) ~[purpur-1.20.6.jar:git-Purpur-2199]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
[14:16:46 INFO]: [ShatteredScrolls] Disabling ShatteredScrolls v2.0.0 (fb1a98d)
```

![image](https://github.com/ShatteredSoftware/ShatteredScrolls/assets/45578026/1750e8ef-3108-4ebc-977e-f2ac1e00abcb)

**Comment by hhenrichsen** [comment]
Closed by release 1.6.0, although I also added some improvements to issues I found during testing (so I recommend 1.6.2).

**Comment by trentnbauer** [comment]
Hi mate, I've installed the latest version and can confirm the plugin has loaded, though I haven't tested it yet.
